### PR TITLE
Portfolio HTML

### PR DESCRIPTION
**Pull request by Orienta248**
My Portfolio-site

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -12,14 +12,28 @@ body {
         font-weight: 3.125rem;
         margin: 0%;
     }
+
+    p,ol,ul {
```
Comment (chiamakaikeanyi) [inline index.css]: As much as possible, avoid styling elements. Use classes instead

File: index.css [hunk 2]
```diff
@@ -12,14 +12,28 @@ body {
...
+
+    #nav > h3 {
+        text-decoration: underline;
+    }
+
     #header {
-        background-color: #f5f5f5;
-        color: #000000;
+        background-color: #505885;
+        color: white;
```
Comment (chiamakaikeanyi) [inline index.css]: Use #fff instead

File: index.css [hunk 3]
```diff
@@ -12,14 +12,28 @@ body {
...
         text-align: center;
         padding: 0.625rem;
         border: 1px solid #ccc;
         margin-bottom: 1px !important;
```
Comment (chiamakaikeanyi) [inline index.css]: Always avoid `!important`

Comment (chiamakaikeanyi) [inline index.css]: Replace all instances where `px` is used with `rem`

File: index.html [hunk 4]
```diff
@@ -50,8 +50,8 @@ <h4>CONTACTS</h4>
                         <li>Phone: +2349092357088</li>
                     </ul>
                     <p>Reach out via any of my social channels:</p>
-                    <p><a href="https://github.com/Orienta248">Github</a></p>
-                    <p><a href="https://app.slack.com/">Slack</a></p>
+                    <p class="socials"><a href="https://github.com/Orienta248">Github</a></p>
```
Comment (chiamakaikeanyi) [inline index.html]: Add `target="_blank"` to make this open on a new tab. You may also want to include a link to your LinkedIn profile

File: index.html [hunk 5]
```diff
@@ -50,8 +50,8 @@ <h4>CONTACTS</h4>
...
+                    <p class="socials"><a href="https://app.slack.com/">Slack</a></p>
```
Comment (chiamakaikeanyi) [inline index.html]: Is this really needed?

File: index.html [hunk 6]
```diff
@@ -50,8 +50,8 @@ <h4>CONTACTS</h4>
...
                 </div>
             <div class="footer">
                 Copyright &copy; Orienta M.C
```
Comment (chiamakaikeanyi) [inline index.html]: Add a dynamic date to this

File: index.css [hunk 7]
```diff
@@ -12,14 +12,28 @@ body {
         font-weight: 3.125rem;
         margin: 0%;
```
Comment (chiamakaikeanyi) [inline index.css]: Once a value is 0, no unit is supposed to be added. Use 0 instead of 0%

File: index.css [hunk 8]
```diff
@@ -12,14 +12,28 @@ body {
         font-weight: 3.125rem;
```
Comment (chiamakaikeanyi) [inline index.css]: We have standard `font-weight` values e.g. `font-weight: 600;`. Use one of them instead

File: index.css (hunk 3 above)
Comment (Orienta248) [reply index.css]: Noted. I basically used it here to understand what it does.

File: index.html [hunk 9]
```diff
@@ -3,40 +3,59 @@
...
+                        As a Junior Software Developer I am tasked with assisting the development team with all aspects of software design and coding. </p>
+                       <p> My primary role is to learn the codebase, attend design meetings, write basic code, fix bugs, and assist the Development Manager in all design-related tasks.As a knowledge lover and I always want to know how things work underneath,
+                           my responsibilities include:
+                        <ol>
+                                <li>Assisting the Development Manager with all aspects of software design and coding.</li>
+                                <li>Attending and contributing to company development meetings.</li>
+                                <li>Learning the codebase and improving your coding skills.</li>
+                                <li>Writing and maintaining code.</li>
+                                <li>Working on minor bug fixes.</li>
+                                <li>Monitoring the technical performance of internal systems.
```
Comment (CodyGbengz) [inline index.html]: I can't seem to locate a corresponding closing `</li>` tag

File: index.css [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+    h1,h2,h3,h4 {
+        font-weight: 3.125rem;
+        margin: 0%;
+    }
+
+    p,ol,ul {
+        color:#6b717f
+    }
+    
+
```
Comment (CodyGbengz) [inline index.css]: Please be consistent with your indentation and remove all unnecessary spaces.

**Pull request closed without merging**